src/main/java/com/fasterxml/jackson/annotation/JsonProperty.java

```diff
@@ -39,7 +39,7 @@ public enum MyEnum {
 {
     /**
      * Special value that indicates that handlers should use the default
-     * name (derived from method or field name) for property.
+     * name (derived from method or field's first variable name) for property.
```

Comment:
Not needed.

Reply:
I suppose you already know that Oracle has plans to listen to the popular views of the general public and the company has decided to alter the language specifications to remove what linting tools deem "smelly code" where declarations of multiple variables under a single field are simply undesirable.  
  
There are only a handful of individuals who use that style anyways and they rarely complain.  If this change is coming in 20th version, that should take the world roughly 10 years to get it all implemented.  
  
This would mean of course a ton of work for source parsing projects but I'm sure they had ample warning.  
  
I agree with you, I also believe a runtime exception and a little Googling is the best way to figure this out.  Not only does it allow people the time to look at the Jackson source code but it also puts them on a journey to become more familiar with the language specifications.  
  
In the interest of public good, I retract this merge request.  
  
Once again, thank you so much for your time sir.

Reply:
@ememisya Ok thank you and apologies for being bit blunt. I appreciate your help here -- at least I am now aware of potential issue. I think I will want to use your suggestions from here to improve Javadocs. Thanks!